test/list_command.bats

```diff
@@ -26,7 +28,10 @@ teardown() {
   run asdf install dummy 1.0.0
   run asdf install tummy 2.0.0
   run asdf list
-  [ "$(echo -e "dummy\n  1.0.0\nmummy\n  No versions installed\ntummy\n  2.0.0")" == "$output" ]
+  [[ "$output" == "$(echo -e "dummy\n  1.0.0")"* ]]
+  [[ "$output" == *"$(echo -e "dummy-broken\n  No versions installed")"* ]]
+  [[ "$output" == *"$(echo -e "mummy\n  No versions installed")"* ]]
+  [[ "$output" == *"$(echo -e "tummy\n  2.0.0")" ]]
```

Comment:
Why does this test code need to change?

Reply:
Adding the broken plugin caused this test to fail, as the test was expecting an explicit output, which had changed due as the `asdf list` command now includes the broken plugin. I also chose to split over multiple lines to aid readabiliy.